internal/exec/go_getter_utils_test.go

```
@@ -0,0 +1,342 @@
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-getter"
+
+	"github.com/cloudposse/atmos/pkg/schema"
+)
+
+type fakeGitGetter struct{}
```

golangci-lint (1.62.2)

type `fakeGitGetter` is unused

(unused)